docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,80 @@
-2. Then, depending on the build flavor specified in the configuration file, a series of standard commands is run.
-   The default for PHP containers, for example, is running `composer install`.
 
-3. Finally, the build hook from the configuration file is run.
-   The build hook comprises one or more shell commands that you write to finish creating your production code base.
-   That could be compiling Sass files, running a bundler, rearranging files on disk,
-   compiling an application in a compiled language, or whatever else you want.
-   Note that, at this point, all you are able to access is the file system; there are no services or other databases available.
+### Deploy steps
 
-Once all of that is completed, we freeze the file system and produce a read-only container image.
-That image is the final build artifact:
-a reliable, repeatable snapshot of your application, built the way you want, with the environment you want.
+Hold requests
+: The system holds incoming requests so there is no service interruption.
 
-Because container configuration (for both your application and its underlying services) is exclusively based on your configuration files
-and your configuration files are managed through Git,
-we know that a given container has a 1:1 relationship with a Git commit.
-That means builds are always repeatable.
-It also means that if we detect that there are no changes that would affect a given container,
-we can skip the build step entirely and reuse the existing container image, saving a great deal of time.
+Unmount current containers
+: The containers gets disconnected from their file system mounts.
 
-In practice, the entire build process usually takes less than a minute.
+Mount file systems
+: The containers that were disconnected in the previous step, connect the file system to the new containers.
 
-## Deploying the application
+If it's a new branch and doesn't have an existing file system, it gets cloned from the parent branch.
+Note that these shouldn't be directories with code since it would be a security risk.
 
-Deploying the application also has several steps, although they're much quicker.
-
-First, we pause all incoming requests and hold them so that there's no interruption of service.
-Then we disconnect the current containers from their file system mounts, if any,
-and connect the file systems to the new containers instead.
-If it's a new branch and there is no existing file system, we clone it from the parent branch.
-
-We then open networking connections between the various containers,
-but only those that were specified in the configuration files (using the `relationships` key).
-No configuration, no connection.
-That helps with security, as only those connections that are actually needed even exist.
+Expose services
+: The system opens the networking connections between the various containers that were specified in the configuration files (using the `relationships` key).
```

Comment:
I feel like the part after "that" isn't really necessary and could be simplified. More in fitting with explaining the concept here. I think readers here might not know what the `relationships` key is. I'm not sure it's quite so helpful to the concept itself.